lib/mutant/mutator/node/block.rb

```diff
@@ -36,7 +36,35 @@ def mutate_body
           return unless body
           emit(body)
           emit_body_mutations
+
+          mutate_body_receiver
+        end
+
+        # Mutate method send in body scope of `send`
+        #
+        # @return [undefined]
+        #
+        # @api private
+        def mutate_body_receiver
+          return unless n_send?(body)
+
+          emit(s(:send, send_meta.receiver, body_meta.selector, *body_meta.arguments))
+        end
+
+        def send_meta
+          AST::Meta::Send.new(send)
+        end
+        memoize :send_meta
+
+        # Method send from body
+        #
+        # @return [Symbol]
+        #
+        # @api private
+        def body_meta
```

Comment:
I see a conceptual problem with exposing this method to the instance namespace: What if someone calls this method without a prior check if body is of type `:send`? 

It would initialize an `AST::Meta::Send` argument with a node of incorrect type. So I think we should keep this as an lvar inside `mutate_body_receiver`, where we also have the guard.